### PR TITLE
Updating Spectrum Tokens to 6.0.0-beta.18.12

### PR DESCRIPTION
**Pull request by GarthDB**
## Description
The build is breaking so I'm looking what components are having issues and what it will take to fix them.

If it's too much work I'll split it into issues.

## To-do list

- [ ] If my change impacts other components, I have tested to make sure they don't break.
- [ ] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [ ] This pull request is ready to merge.

# ⚠️ DO NOT SQUASH, USE MERGE COMMIT ⚠️

### CODE REVIEW

**Comment by lazd** [comment]
Build fixed, some files were missing.

File: components/pagination/package.json [hunk 1]
```diff
@@ -24,17 +24,16 @@
     "@spectrum-css/vars": "^2.3.0"
   },
   "devDependencies": {
-    "@spectrum-css/actionbutton": "^1.0.0-beta.0",
-    "@spectrum-css/button": "^3.0.0-beta.4",
-    "@spectrum-css/component-builder": "^1.2.1-beta.0",
-    "@spectrum-css/icon": "^3.0.0-beta.0",
-    "@spectrum-css/splitbutton": "^3.0.0-beta.4",
-    "@spectrum-css/textfield": "^3.0.0-beta.4",
+    "@spectrum-css/button": "^3.0.0-beta.5",
```
Comment (lazd) [inline components/pagination/package.json]: This is incorrect, was supposed to be action button as a dep. Will fix.

File: components/quickaction/package.json [hunk 2]
```diff
@@ -21,14 +21,14 @@
     "@spectrum-css/vars": "^2.3.0"
   },
   "devDependencies": {
-    "@spectrum-css/actionbutton": "^1.0.0-beta.0",
-    "@spectrum-css/checkbox": "^3.0.0-beta.4",
-    "@spectrum-css/component-builder": "^1.2.1-beta.0",
+    "@spectrum-css/button": "^3.0.0-beta.5",
```
Comment (lazd) [inline components/quickaction/package.json]: This was supposed to be actionbutton, will fix

**Review by jianliao — APPROVED** [review]
LGTM

**Pull request merged**